go/vt/vtgate/planbuilder/testdata/filter_cases.json

```diff
@@ -2679,10 +2702,10 @@
   },
   {
     "comment": "select * from samecolvin where col = :col",
-    "query": "select * from samecolvin where col = :col",
+    "query": "select * from samecolvin where col = 'val'",
```

Comment:
same.

Reply:
thanks , restored the test case with https://github.com/vitessio/vitess/pull/17342/commits/b99ebabe960b49792dd426509d68fbad73a3fda3